paper-input-error.html

```diff
@@ -50,7 +51,7 @@
     </style>
 
     <content></content>
-
```

Comment:
the empty line? No, it's my editor being "tabs not spaces" or something. I didn't touch it.